test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -5440,6 +5440,52 @@ public void PackCommand_PackageReadmeFile_ReadmeFileIsEmpty_Fails()
             }
         }
 
+        [PlatformFact(Platform.Windows)]
+        public void PackCommand_PackageReadmeFile_BasicFunc_WithSymbol_Succeeds()
+        {
+            var testDirBuilder = TestDirectoryBuilder.Create();
+            var projectBuilder = ProjectFileBuilder.Create();
+
+            testDirBuilder
+                .WithFile("test\\readme.md", 10)
+                .WithFile("test\\folder\\notes.txt", 10)
+                .WithFile("test\\folder\\nested\\content.txt", 10)
+                .WithFile("test\\folder\\nested\\sample.txt", 10)
+                .WithFile("test\\icon.jpg", 10)
+                .WithFile("test\\other\\files.txt", 10)
+                .WithFile("test\\utils\\sources.txt", 10);
+
+            projectBuilder
+                .WithProjectName("test")
+                .WithProperty("IncludeSymbols", "true")
+                .WithProperty("SymbolPackageFormat", "snupkg")
+                .WithPackageReadmeFile("readme.md")
+                .WithItem(itemType: "None", itemPath: "readme.md", packagePath: string.Empty, pack: "true")
+                .WithItem(itemType: "None", itemPath: "other\\files.txt", packagePath: null, pack: "true")
+                .WithItem(itemType: "None", itemPath: "folder\\**", packagePath: "media", pack: "true")
+                .WithItem(itemType: "None", itemPath: "utils\\*", packagePath: "utils", pack: "true");
+
+            using (var srcDir = msbuildFixture.Build(testDirBuilder))
+            {
+                projectBuilder.Build(msbuildFixture, srcDir.Path);
+                var result = msbuildFixture.PackProject(projectBuilder.ProjectFolder, projectBuilder.ProjectName, string.Empty);
```

Comment:
explicit type

Reply:
Assert for `result.Success`?. ..or otherwise no need for variable.

Reply:
Thanks! Addressed.
There are assertions at https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs#L270-L271

Reply:
Oh gotcha, thanks!